SDWebImage/Core/UIButton+WebCache.h

```diff
@@ -343,7 +343,7 @@
                     placeholderImage:(nullable UIImage *)placeholder
                              options:(SDWebImageOptions)options
                             progress:(nullable SDImageLoaderProgressBlock)progressBlock
-                           completed:(nullable SDExternalCompletionBlock)completedBlock;
+                           completed:(nullable SDExternalCompletionBlock)completedBlock NS_SWIFT_NAME(sd_setBackgroundImage(with:state:placeholder:options:progress:completed:));
```

Comment:
I think `progress` name doesn't sound like a name for a closure, it can be confused with `Foundation.Progress` class. Maybe `progressBlock` would be more readable?

Reply:
I think progress LGTM. AFNetworking's api also have `progress` [code reference](https://github.com/AFNetworking/AFNetworking/blob/c646ed658cb675304ff80db60bc7175251407c1c/AFNetworking/AFHTTPSessionManager.m#L123)

Reply:
I also think `progress` LGTM.